src/sentry/static/sentry/app/views/discover/aggregations/aggregation.tsx

```diff
@@ -94,6 +94,7 @@ export default class AggregationRow extends React.Component<
   inputRenderer = (props: AggregationProps) => {
     const onChange = (evt: any) => {
       if (evt && evt.target && evt.target.value === '') {
+        evt.persist();
```

Comment:
Should components downstream even need the full event object?

Reply:
The downstream usage of the event is react-select ☠️